### PR TITLE
Remove short term sitenotice that uploading is available

### PR DESCRIPTION
**Pull request by dmehus**
Several announcements on Discord together with this short sitenotice and the fact that actually uploading files will make it clear that uploads are again available. No need for more than a few hour sitenotice.

### CODE REVIEW

File: Sitenotice.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-if ( $wmgSiteNoticeOptOut ) {
+/* if ( $wmgSiteNoticeOptOut ) {
```
Comment (Universal-Omega) [inline Sitenotice.php:3]: Should not comment out this if statement.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: Sitenotice.php (hunk 1 above)
Comment (dmehus) [reply Sitenotice.php:3]: @Universal-Omega Oh okay, where do you want me to move it to then?

Comment (Universal-Omega) [reply Sitenotice.php:3]: To only the second portion where the sitenotice actually is.

Comment (dmehus) [reply Sitenotice.php:3]: > To only the second portion where the sitenotice actually is.

Okay, doing that now, and hoping I get it right.

Comment (dmehus) [reply Sitenotice.php]: 👍 **Done**

**Pull request merged**